### DISCUSSION TITLE
Cloudlog and upload to EQSL

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#1639] [Q&A] [answered]
Can someone shed a light on this? It seems I cannot upload my QSO's /P to EQSL for some reason. 

Let's assume I want to upload PAFF-0162 to EQSL. How would I go about?

Thanks for info.

Regards Rob

<table>
<tr><td></td><td>Account</td><td>Station profile name</td><td>EQSL Nick</td></tr>
<tr><td>Cloudlog</td><td>PD0RZH</td><td>Delft</td><td>Home QTH</tr>
<tr><td>Cloudlog</td><td>PD0RZH/P</td><td>Meijendel</td><td>PAFF0162</tr>
<tr><td>Cloudlog</td><td>PD0RZH/P</td><td>Manteling van Walcheren</td><td>Vrouwenpolder</tr>
<tr><td></td></tr>
<tr><td>EQSL</td><td>PD0RZH</td><td></td><td>Home QTH</td></tr>
<tr><td>EQSL</td><td>PD0RZH/P</td><td></td><td> PAFF0162</td></tr>
<tr><td>EQSL</td><td>PD0RZH/P</td><td></td><td>Vrouwenpolder</td></tr>
</table>

**Comment by magicbug** [comment]
Not best to answer this as I don't fully get eQSLs upload system however @nolith did quite a bit of work on the dev branch and it seems to be working well so perhaps better placed to explain how to set things up!

**Comment by nolith** [comment, marked as answer]
Hello 👋 are you running the development branch or master?

**Reply by PA4RB** [reply]
This is on Master.

Thanks

**Reply by nolith** [reply]
ok, I'm sorry but there is no way to have this working on `master`. I implemented that feature on the development branch `station_logbooks`.

I'm not telling you to run the development version, but if you do so then the eQSL logic will be as follow.

- The password is still the one in your cloudlog account profile
- Each location you define with an eQSL 4th nickname will now partake in eQSL upload and download independently. **using the location callsign as eqsl username**, the qth nickname as the location identifier, and your global password as password

I hope this helps

**Reply by PA4RB** [reply]
Thanks for your explanation Alessio. Although I have downloaded and installed the dev version on a MAMP instance I will wait for the next 2.0 production. 

73 Rob
